### PR TITLE
Feature/skeleton

### PR DESCRIPTION
**Pull request by DmitryOnopriienko**

### CODE REVIEW

File: src/main/kotlin/com/ajaxproject/warehouse/entity/Customer.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    @Column(name = "first_name")
+    var firstName: String,
+    var surname: String,
+    var patronymic: String?,
+    var email: String,
+
+    @Column(name = "phone_number")
+    var phoneNumber: String?,
+    var birthday: Date?,
+    var comment: String?)
```
Comment (ajax-shulepa-a) [inline src/main/kotlin/com/ajaxproject/warehouse/entity/Customer.kt]: ) should be on a new line
file should end with an empty line
fix in all places

**Review by ajax-shulepa-a — APPROVED** [review]

**Review by ajax-shulepa-a — CHANGES_REQUESTED** [review]
see comment (approved by mistake)

File: src/main/kotlin/com/ajaxproject/warehouse/entity/Customer.kt (hunk 1 above)
Comment (DmitryOnopriienko) [reply src/main/kotlin/com/ajaxproject/warehouse/entity/Customer.kt]: Fixed

**Review by ajax-shulepa-a — CHANGES_REQUESTED** [review]
files must end with an empty line (see knowledge sharing topic in tg)

**Review by ajax-shulepa-a — APPROVED** [review]

**Pull request merged**